### PR TITLE
Nerfs cakehat

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
It is with a heavy heart that I must do this

### CODE REVIEW

**Comment by Armhulen** [comment]
you don't have you you just want to :-1:

**Comment by Core0verload** [comment]
I like the nerfs that way. Could you take a same approach on slips?

**Comment by Mercenaryblue** [comment]
I see right through you!

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -44,10 +44,11 @@
  */
 /obj/item/clothing/head/hardhat/cakehat
 	name = "cakehat"
-	desc = "You put the cake on your head. Brilliant."
+	desc = "You put the cake on your head. Brilliant. It keeps getting stronger somehow..."
```
Comment (phil235) [inline code/modules/clothing/head/misc_special.dm]: no meme shit please

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 2]
```diff
@@ -167,6 +167,7 @@
 	name = "high frequency blade"
 	desc = "Bad references are the DNA of the soul."
 	attack_verb = list("chopped", "sliced", "cut", "zandatsu'd")
+	hitsound = 'sound/weapons/bladeslice.ogg'
```
Comment (coiax) [inline code/game/objects/items/weapons/holy_weapons.dm]: STEALH BUFF/NERF TO CHAPLAIN, UNDOCUMENTED, LYNCH

**Comment by KorPhaeron** [comment]
@phil235 

Okay

**Pull request merged**